source/common/router/header_parser.cc

```diff
@@ -55,7 +54,7 @@ HeaderFormatterPtr parseInternal(const envoy::config::core::v3::HeaderValue& hea
 
   absl::string_view format(header_value.value());
   if (format.empty()) {
-    return std::make_unique<PlainHeaderFormatter>("", append);
```

Comment:
For my education, why did we ever have append here?

Reply:
I see actually, you now propagate this independent of the header formatter, seems cleaner.